.github/integration/tests/10_encrypt_decrypt.sh

```diff
@@ -0,0 +1,125 @@
+#!/bin/bash
+set -e
+
+# Function checking that a file was encrypted
+function check_encypted_file() {
```

Comment:
This function is not only checking "one file" but checking all files separated by white spaces provided in the first argument. 
Note sure if this is best. I would expect the function is called as 
```
check_encrypted_file file1 file2
```
Instead of 
```
check_encrypted_file "file1 file2"
```

Suggested changes:
1. Update the comment line, indicating that it tries to checking all files provided 
2. Fixing the typo of the function name, i.e. `check_encypted_file` to `check_encrypted_file`
3. Probably change the function to 
e.g. 
```sh
function check_encrypted_file() {
    for k in "$@"; do
        output=$(head -c 8 "$k")

        if [[ "$output" == "crypt4gh" ]]; then
            echo "Encrypted data file: $k"
        else
            echo "Failed to encrypt file: $k"
            exit 1
        fi
    done
}
```
so that the arguments of a list of files are provided in a more natural way.

Reply:
Fixed